README.md

```diff
@@ -117,12 +117,16 @@ It is recommended that developers create a virtual environment for local Python
 
     python3 -m venv venv_festerize
     . venv_festerize/bin/activate
-    pip install -e . pytest
+    pip install -e . black pytest
 
 To run the tests:
 
     pytest
 
+Before committing, make sure you format all the Python source files:
```

Comment:
Might be nice to supply a pre-push git hook?

Reply:
Good idea. I created a pre-push hook so a dev could keep the files in whatever state is most appropriate for them until they're ready to share

Reply:
Yeah, that's better.